test/images/README.md

```diff
@@ -54,26 +54,33 @@ to build the image and push it to your own registry first, and run some tests th
 For these steps, see the sections below.
 
 After the desired changes have been made, the affected images will have to be built and published,
-and then tested. After the pull request with those changes has been approved, the new images will be
-built and published to the `gcr.io/kubernetes-e2e-test-images` registry as well.
-
-Currently, the image building process has been automated with the Image Promoter, but *only* for the
-Conformance images (`agnhost`, `jessie-dnsutils`, `kitten`, `nautilus`, `nonewprivs`, `resource-consumer`,
-`sample-apiserver`).  After the pull request merges, a postsubmit job will be started with the new changes,
-which can be tracked [here](https://testgrid.k8s.io/sig-testing-images#post-kubernetes-push-images).
-After it passes successfully, the new image will reside in the `gcr.io/k8s-staging-e2e-test-images/${IMAGE_NAME}:${VERSION}`
-registry, from which it will have to be promoted by adding a line for it
-[here](https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-e2e-test-images/images.yaml).
-For this, you will need the image manifest list's digest, which can be obtained by running:
+and then tested. After the pull request with those changes has been approved and merged, a postsubmit
+job will then be triggered which will build the images that were changed (for example, if a change was
+made in `test/images/agnhost`, then the job [post-kubernetes-push-e2e-agnhost-test-images](
+https://testgrid.k8s.io/sig-testing-images#post-kubernetes-push-e2e-agnhost-test-images)
+ will be triggered). The image will then be pushed to the `gcr.io/k8s-staging-e2e-test-images` registry,
+from which it can then promoted to the `k8s.gcr.io/e2e-test-images` registry by adding a line in
+[kubernetes/k8s.io](https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-e2e-test-images/images.yaml). For this, you will need the image manifest list's digest, which can be obtained by running:
 
 ```bash
 manifest-tool inspect --raw gcr.io/k8s-staging-e2e-test-images/${IMAGE_NAME}:${VERSION} | jq '.[0].Digest'
 ```
 
+All the postsubmit jobs and their logs for all the images can be seen in
+[testgrid](https://testgrid.k8s.io/sig-testing-images).
+
 The images are built through `make`. Since some images (e.g.: `busybox`) are used as a base for
 other images, it is recommended to build them first, if needed.
 
 
+### Creating and promoting new images
+
+If you intend to add an entirely different image and have it automatically built by the Image Builder
+and used in E2E tests, you will also have to define the postsubmit prow job for it. This can easily
+be done by running [this script](https://github.com/kubernetes/test-infra/blob/10a290aaa9f2657ec46d8a2a50e0a3eeca5b1715/config/jobs/image-pushing/k8s-staging-e2e-test-images.sh)
```

Comment:
other links are set to the `master` branch should this one be linked to that as well?